.github/workflows/ci.yml

```diff
@@ -53,3 +53,11 @@ jobs:
       - run: python test.py dist/
         env:
           PYTHONUTF8: 1
+  typecheck:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+      - run: pip install pyrefly
+      - name: Typecheck
+        run: pyrefly check -c pyproject.toml
```

Comment:
This can be changed to `pyrefly check` once https://github.com/facebook/pyrefly/issues/27 lands